index.html

```diff
@@ -50,7 +53,179 @@ <h2>Software Developer</h2>
         need coded, don’t hestiate to contact me.
       </p>
     </section>
+    <!--headline sections ends here-->
 
-    <hr id="horizontal-line" />
+    <hr class="horizontal-line" />
+
+    <!--about sections-->
+    <section class="about-section">
+      <div>
+        <h3 class="title">Languages</h3>
+        <ul class="skills">
+          <li>Javascript</li>
+          <li>Python</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </div>
```

Comment:
- [ ] About section has a background image in the original design. To stick to the design please add it.
You can see the top-right side white circle.
![image](https://user-images.githubusercontent.com/38635158/148599287-334a77ee-f47b-4587-aa1c-0eeb42f82ba3.png)

Reply:
background image added successfully